CMakeLists.txt

```diff
@@ -34,6 +28,7 @@ set(CMAKE_SHARED_MODULE_PREFIX "")
 set(CMAKE_SKIP_PREPROCESSED_SOURCE_RULES TRUE)
 set(CMAKE_SKIP_ASSEMBLY_SOURCE_RULES TRUE)
 set(CMAKE_COLOR_MAKEFILE OFF)
+set(CMAKE_POSITION_INDEPENDENT_CODE OFF)
```

Comment:
Previously we only turned this off for ASM libraries. Any difference to be expected from doing it globally?

Reply:
Far as i'm aware there's no position independent code on windows and -fPIC being added for asm files was a bug
Might be possible to remove that line but i've not tried it, it could also be set as a target property, i *think* its just freeldr that had issues with -fPIC